modules/end-user-guide/partials/proc_allowing-users-to-define-workspace-creation-strategy.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// creating-a-workspace-from-a-remote-devfile
+
+[id="allowing-users-to-define-workspace-creation-strategy_{context}"]
+= Allowing users to define workspace creation strategy
+
+
+This section describes how to control the strategy of creating a {prod-short} workspace using a factory URL. {prod-short} currently supports the following options: 
```

Comment:
Instead of mentioning the document ("This section..."), we start with a brief abstract or summary of the use case. For example, without a better understanding the use case here, I'd say something like, "As a developer, you can configure {prod-short} to create a new workspace each time it accepts a factory URL, or to reuse the existing workspace if a user already has one."
Then, you can go on to describe the options in more detail, like you've done here.

Reply:
IBM SG, pg 22
Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes".
Instead, focus on the purpose or value of the content, and include words that users are likely to search
for. For example, in a short description in an online help topic, provide users with enough information to
determine whether the topic meets their needs. You can use self-referential text in learning modules
and tutorials.

Reply:
The need for a short description or abstract is well-established in topic-based writing standards, such as DocBook and DITA, but is only now being formalized at Red Hat: https://github.com/redhat-documentation/supplementary-style-guide/issues/75